src/bits.rs

```diff
@@ -16,6 +16,17 @@ use core::{
     ops::{BitAnd, BitAndAssign, BitOr, BitOrAssign, BitXor, BitXorAssign, Not},
 };
 
+pub(crate) fn trailing_zeros_large(words: &[Word]) -> usize {
```

Comment:
Call this `trailing_zeros` and it should work for `words.len() == 1` (no reason not to).

Reply:
Done in https://github.com/tczajka/ibig-rs/pull/11/commits/a66b5164f35520c67b6bf825044f750537690e88